src/app/containers/MostRead/section.test.jsx

```diff
@@ -4,7 +4,9 @@ import MostReadSection from './section';
 
 describe('MostReadSectionLabel assertion', () => {
   it('should render most-read section with correct attributes', async () => {
-    const { container } = render(<MostReadSection />);
+    const { container } = render(
+      <MostReadSection>Some child content</MostReadSection>,
```

Comment:
This needs to be added following a required prob being added to the `MostReadSection` - was this intentional @simonsinclair https://github.com/bbc/simorgh/blob/4bfab21ca108f7d5a88bb1fb6e29149f73ab94ce/src/app/containers/MostRead/section.jsx#L12

Reply:
Yes, thanks for doing that.